github-templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +15,9 @@ Linked PRs: (links to corresponding PRs, optional)
 ## TODOs:
 - [ ] +1
 - [ ] Passes tests
+- [ ] "How to test" steps have been run through
```

Comment:
Ah, good point. Yeah the +1 essentially means that the reviewer has ran those steps. I'll remove.